config/build/rollup.config.js

```diff
@@ -0,0 +1,324 @@
+/* eslint-disable import/no-commonjs */
+import fs from "fs";
+import path from "path";
+import autoExternal from "rollup-plugin-auto-external";
+import {babel} from "@rollup/plugin-babel";
+import {terser} from "rollup-plugin-terser";
+import copy from "rollup-plugin-copy";
+import resolve from "@rollup/plugin-node-resolve";
+import replace from "@rollup/plugin-replace";
+import filesize from "rollup-plugin-filesize";
+import {preserveShebangs} from "rollup-plugin-preserve-shebangs";
+import rollupExecutable from "rollup-plugin-executable-output";
+
+const createBabelPresets = require("./create-babel-presets.js");
+const createBabelPlugins = require("./create-babel-plugins.js");
+
+/**
+ * We support the following config args with this rollup configuration:
+ *
+ * --configPackages
+ *      A comma-delimited list of package names to build.
+ *      The "perseus-" portion can be omitted.
+ *      Default: All packages.
+ *
+ * --configFormats
+ *      A comma-delimited list of formats to build.
+ *      Valid options are "cjs" and "esm".
+ *      Default: cjs, esm
+ *
+ * --configPlatforms
+ *      A comma-delimited list of platforms to build.
+ *      Valid options are "browser" and "node".
+ *      Default: browser, node
+ *
+ * --configEnvironment
+ *      A string to use as the NODE_ENV environment variable.
+ *      Valid options are "development" and "production".
+ *      Default: We do not target an environment so that consumers can benefit
+ *               from the default behavior.
+ */
+
+/**
+ * Make path to a package relative path.
+ */
+const makePackageBasedPath = (pkgName, pkgRelPath) =>
+    path.normalize(path.join("packages", pkgName, pkgRelPath));
+
+/**
+ * Generate the rollup output configuration for a given
+ */
+const createOutputConfig = (pkgName, format, targetFile) => ({
+    file: makePackageBasedPath(pkgName, targetFile),
+    sourcemap: true,
+    format,
+});
+
+/**
+ * Get a set of strings from a given string, returning the defaults
+ *
+ * This assumes comma-delimited strings.
+ */
+const getSetFromDelimitedString = (arg, defaults) => {
+    const values =
+        arg != null && arg.length > 0
+            ? arg
+                  .split(",")
+                  .map((p) => p.trim())
+                  .filter(Boolean)
+            : [];
+
+    return new Set(values.length ? values : defaults);
+};
+
+/**
+ * Determine what platforms we are targetting.
+ */
+const getPlatforms = ({configPlatforms}) =>
+    getSetFromDelimitedString(configPlatforms, ["browser", "node"]);
```

Comment:
I think we could probably remove `"node"` from here and update the package and remove "module" and "browser" fields from the package.json files.